ihatemoney/forms.py

```diff
@@ -245,7 +230,7 @@ class BillForm(FlaskForm):
     original_currency = SelectField(
         _("Currency"),
         choices=[
-            (currency_name, currency_name)
+            (currency_name, render_localized_currency(currency_name, detailed=False))
```

Comment:
Is there anything that prevents using the detailed format here?

If we switch to something like select2 or equivalent, it would allow to search by name (e.g. "zloty").

That being said, I agree that this field will rarely be used.

Reply:
When a user is selecting a currency for one bill, he/she should be aware of the currency, and I don't think there is a need to clutter the UI with a lot of text. We will see if we use a filtering select widget in the future.